app/src/main/java/org/permanent/permanent/ui/activities/MainActivity.kt

```diff
@@ -215,7 +216,10 @@ class MainActivity : PermanentBaseActivity(), Toolbar.OnMenuItemClickListener {
                     return@setNavigationItemSelectedListener false
                 }
 
-                R.id.logOut -> viewModel.deleteDeviceToken()
+                R.id.logOut -> {
+                    viewModel.deleteDeviceToken()
+                    EventsManager(this).resetUser()
```

Comment:
There are multiple places from where the user is logged out. Maybe should add this there too.